mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -257,15 +281,16 @@ def StopTask(self, taskId=None, taskIds=None, taskType=None, taskTypes=None, cyc
             taskTypes (list[str], optional): If specified, a list of task types to stop.
             cycleIndex (str, optional): Unique cycle index string for tracking, backing up, and differentiating cycles.
             waitForStop (bool, optional): If True, then wait for task to stop, otherwise just trigger it to stop, but do not wait (Default: True)
-            removeTask (bool, optional): If True, then remove the task from being tracked by the vision manager and destroy all its resources. Will wait for the task to end before returning.
-            fireandforget (bool, optional): If True, does not wait for the command to finish and returns immediately. The command remains queued on the server.
+            removeTask (bool, optional): If True, then remove the task from being tracked by the vision manager and destroy all its resources. Will wait for the task to end before returning. (Default: False)
+            fireandforget (bool, optional): If True, does not wait for the command to finish and returns immediately. The command remains queued on the server. (Default: False)
             timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 2.0)
 
         Returns:
             dict: A dictionary with the structure:
 
-            isStopped (bool): true, if the specific taskId or set of tasks with a specific taskType(s) is stopped
+                - isStopped (bool): true, if the specific taskId or set of tasks with a specific taskType(s) is stopped
         """
+        log.verbose('Stopping detection thread...')
```

Comment:
Can you put these additional logs into a separate MR as well? It's not clear that they won't be spammy.